package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "disciple-tools",
   "version": "0.1.0",
-  "lockfileVersion": 3,
```

Comment:
@CptHappyHands the package-lock.json is in the .gitignore and should not be committed.